src/lib/y2partitioner/widgets/blk_devices_table.rb

```diff
@@ -197,6 +195,96 @@ def end_value(device)
         return "" if device.region.empty?
         device.region.end
       end
+
+      DEVICE_ICONS = {
+        disk:      Icons::HD,
+        dasd:      Icons::HD,
+        multipath: Icons::MULTIPATH,
+        partition: Icons::HD_PART,
+        raid:      Icons::RAID,
+        lvm_vg:    Icons::LVM,
+        lvm_lv:    Icons::LVM_LV
+      }
+
+      # Table icon for the device
+      #
+      # @see DEVICE_ICONS
+      #
+      # @param device [Y2Storage::BlkDevice]
+      # @return [Yast::Term] icon
+      def type_icon(device)
+        return type_icon(device.plain_blk_device) if device.is?(:lvm_pv)
+
+        type = DEVICE_ICONS.keys.find { |k| device.is?(k) }
+        icon = type.nil? ? Icons::DEFAULT_DEVICE : DEVICE_ICONS[type]
+
+        small_icon(icon)
+      end
+
+      DEVICE_LABELS = {
+        disk:          _("DISK"),
```

Comment:
If you use `_("DISK")` here, at class definition level, these strings will not change in the system language is changed at some later point, when the class is already defined. Check how `_N("DISK")` is used in other parts with the same problem.